src/createShots.ts

```diff
@@ -321,11 +342,14 @@ export const createShots = async () => {
     );
   }
 
-  return [
-    ...storybookShotItems,
-    ...pageShotItems,
-    ...ladleShotItems,
-    ...histoireShotItems,
-    ...customShotItems,
-  ];
+  return {
+    shotItems: [
+      ...storybookShotItems,
+      ...pageShotItems,
+      ...ladleShotItems,
+      ...histoireShotItems,
+      ...customShotItems,
+    ],
```

Comment:
As a code suggestion, I believe we could have a `const shotItems` at the beginning of the function and `.push` after each is obtained, so we don't have this here at the bottom.

It would also be good if each of them were in their own function as this function is quite big